src/legacy/ui/ui_render/bootstrap/template.js.hbs

```diff
@@ -78,17 +96,13 @@ if (window.__kbnStrictCsp__ && window.__kbnCspNotEnforced__) {
           '{{this}}',
         {{/each}}
         '{{regularBundlePath}}/commons.bundle.js',
-        {{!-- '{{regularBundlePath}}/plugin/data/data.plugin.js', --}}
         '{{regularBundlePath}}/plugin/kibanaUtils/kibanaUtils.plugin.js',
         '{{regularBundlePath}}/plugin/esUiShared/esUiShared.plugin.js',
-        '{{regularBundlePath}}/plugin/kibanaReact/kibanaReact.plugin.js'
-    ], function () {
-      load([
+        '{{regularBundlePath}}/plugin/kibanaReact/kibanaReact.plugin.js',
         '{{regularBundlePath}}/{{appId}}.bundle.js',
         {{#each styleSheetPaths}}
           '{{this}}',
         {{/each}}
-      ])
-    });
```

Comment:
Since we're now using `async=false` I don't think we need to wait for the first array of scripts to load before we start loading the entry point. This is because these scripts should always execute in order they are put on the page, not in load order. Let me know if I'm wrong.

Reply:
I know @tylersmalley encountered issues with IE11 regarding this loading order. But I think this change goes in the right direction?

Reply:
I put this back for now.